### ISSUE TITLE
LORE finetuning gives wierd bounding boxes.

### ISSUE THREAD

**Issue by LucaDeGrandis** [AlibabaResearch/AdvancedLiterateMachinery#23]
Hello, thank you for the great work you did with LORE.

I am trying finetuning the small version of LORE on a custom dataset but it seems that when I do the model results are getting worst.
I tried some zero-shot inference with ckpt_ptn model weights.
For finetuning I used
```
!python main.py ctdet_small \
	--dataset table_small \
	--exp_id CTU_fold_0 \
	--dataset_name ctu_fold_0 \
	--image_dir /datadrive/images/CTU \
	--wiz_2dpe \
	--wiz_stacking \
	--tsfm_layers 3 \
	--stacking_layers 3 \
	--batch_size 16 \
	--master_batch 12 \
	--arch dla_34 \
	--lr 1e-4 \
	--K 500 \
	--MK 1000 \
	--num_epochs 10 \
	--lr_step '100, 160' \
	--gpus 0 \
	--num_workers 16 \
	--val_intervals 1 \
	--save_all \
	--load_model /datadrive/LORE_experiments/pretrined_checkpoints/small_model_best.pth \
	--load_processor /datadrive/LORE_experiments/pretrined_checkpoints/small_processor_best.pth
```
where small_model_best.pth and small_processor_best.pth are the names I used to save the ckpt_ptn model weights.

I also produced annotations similar to the ones you provided in the subset of PubTabNet. I can provide an example if necessary.

Am I doing something wrong in the script?
Thank in advance!

**Comment by BrownXing** [comment]
Thank you. We have fixed some issue in the training scripts, please try the latest version. If the problem persists, feel free to contact us!

**Comment by Saeed11b95** [comment]
Hi @BrownXing 
Are the scripts to convert datasets like Pubtabnet or Fintabnet included in this repo, I can't seem to find them. I mean the scripts to convert Jsonl/HTML annotations to coco with logic_axis. Thanks

**Comment by BrownXing** [comment]
> Hi @BrownXing Are the scripts to convert datasets like Pubtabnet or Fintabnet included in this repo, I can't seem to find them. I mean the scripts to convert Jsonl/HTML annotations to coco with logic_axis. Thanks

We have not release such a script in this repo, but you could get the logical axis from HTLM labels by first preorder traversing the HTML sequence and computing the logical axis according to the hierarchy and the row_span/column_span attributes of <td>.
